packaging/release

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env node
+
+/* eslint-disable import/no-extraneous-dependencies, no-console */
+const git = require('simple-git/promise');
+const util = require('util');
+const exec = util.promisify(require('child_process').exec);
+const { argv: { branch, platform, token } } = require('yargs');
+const checkVersion = require('./check-version');
+const updateRemoteVersion = require('./update-remote');
+
+const release = async () => {
+  const track = branch === 'dev' ? 'alpha' : 'beta';
+
+  // Get all tags, sort in desc order, and get latest from current track
+  const gitTags = await git().tags();
+  const gitTag = gitTags.all.reverse().find(tag => tag.indexOf(track) > -1);
+  console.log(`Last Git tag: ${gitTag}`);
+
+  // Get a new release version and update package.json
+  const releaseVersion = checkVersion(branch, gitTag);
+  console.log(`New release version: ${releaseVersion}`);
+
+  console.log('TODO: stream output for the next command');
```

Comment:
can you elaborate a bit?

Reply:
Line 26 is only going to log to the console once `npm run dist:platform` is done in the current way, so anyone watching will be waiting a while. In the future, we should use `spawn` instead of `exec` from `child_process`

Reply:
That allows you to pipe the output from the command to `console.log`